sdk/storagemover/Azure.ResourceManager.StorageMover/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("StorageMover")]
```

Comment:
I assume here should be `Microsoft.StorageMover`

Reply:
Done